CHANGELOG.md

```diff
@@ -77,23 +77,23 @@
 ### ❌ Removed
 
 ## stream-chat-android-compose
-- Added `StreamDimens` option to the `ChatTheme`, to allow for dimension customization across the app.
-- Added localization support for the components related the channel list.
-- Added the `emptySearchContent` parameter to `ChannelList` component that allows to customize the empty placeholder, when there are no channels matching the search query.
-- Fixed channel options that are displayed in the `ChannelInfo` component.
-- Added support for the muted channel indicator in the message list.
-- Added `ChannelNameFormatter` option to the `ChatTheme`, to allow for channel name format customization across the app.
-
 ### 🐞 Fixed
+- Fixed channel options that are displayed in the `ChannelInfo` component.
 
 ### ⬆️ Improved
 - Improved the icon set and polished the UI for various Messages features
 - Improved the set of customization options for the `DefaultChannelItem`
 - Updated documentation for Channels set of features
+- Now it is possible to search for distinct channels by member names using `ChannelListViewModel`.
 
 ### ✅ Added
 - Added a new parameter to the `AttachmentFactory` called `previewContent` that represents attachments within the MessageInput
 - Added the `leadingContent`, `detailsContent`, `trailingContent` and `divider` Slot APIs for the `DefaultChannelItem`
+- Added `StreamDimens` option to the `ChatTheme`, to allow for dimension customization across the app.
+- Added localization support for the components related the channel list.
+- Added the `emptySearchContent` parameter to `ChannelList` component that allows to customize the empty placeholder, when there are no channels matching the search query.
```

Comment:
I'm wondering if all this is already merged to develop? Feels like something that shouldn't show up in this git diff

Reply:
It was moved from ~10 lines up where it wasn't in any section under Compose into the "Added" section

Reply:
Ohh okay yeah makes sense - I'm blind 👁️